### DISCUSSION TITLE
单窗口模式下标签页重叠

### DISCUSSION THREAD

**Discussion by loikuan** [ywmoyue/biliuwp-lite#196] [Q&A]
单窗口模式下打开三级或以上界面时会与上一界面重叠
![屏幕截图 2023-07-10 135528](https://github.com/ywmoyue/biliuwp-lite/assets/128059419/b13e7279-f9be-4eda-a500-d90a35b7a3ac)
![屏幕截图 2023-07-10 135537](https://github.com/ywmoyue/biliuwp-lite/assets/128059419/0530d524-ba1a-4e44-9267-ad7d3fef2fdb)

**Comment by ywmoyue** [comment]
已处理了，下一版本发 https://github.com/ywmoyue/biliuwp-lite/issues/192

**Comment by ywmoyue** [comment]
4.6.7已修复，麻烦验证一下
